cli.h

```diff
@@ -83,4 +84,44 @@ void InsertStudent(vector<Student>& students)
     temp.isStudy = true;
 
     students.push_back(temp);
+}
+void Export_File(vector<Student>& students) {
```

Comment:
Не обязательно применять в аргументах ссылочный тип, в данном случае не обязательно.
```suggestion
void Export_File(vector<Student> students) {
```